src/main/java/com/avispl/dal/communicator/cisco/CiscoCommunicator.java

```diff
@@ -1569,6 +1588,151 @@ private void populateRoomAnalyticsData(Map<String, String> statistics, List<Adva
+                    addStatisticsParameter(typedStats, idKey, mergeAndNormalizeStrings(typedStats.get(idKey), connectedDevice.getID(), "; "));
+                    addStatisticsParameter(typedStats, nameKey, mergeAndNormalizeStrings(typedStats.get(nameKey), connectedDevice.getName(), "; "));
+                    addStatisticsParameter(typedStats, serialNumberKey, mergeAndNormalizeStrings(typedStats.get(serialNumberKey), connectedDevice.getSerialNumber(), "; "));
+                    addStatisticsParameter(typedStats, softwareInfoKey, mergeAndNormalizeStrings(typedStats.get(softwareInfoKey), connectedDevice.getSoftwareInfo(), "; "));
+                    addStatisticsParameter(typedStats, typeKey, mergeAndNormalizeStrings(typedStats.get(typeKey), connectedDevice.getType(), "; "));
+                    addStatisticsParameter(typedStats, key + PROPERTY_TOTAL_DEVICES_COUNT, String.valueOf(++totalDevicesOfStateAndType));
+                });
+
+                disconnectedTypedStats.values().forEach(statistics::putAll);
+                connectedTypedStats.values().forEach(statistics::putAll);
+            }
+        }
+
+        PeripheralsConfiguration peripheralsConfiguration = configuration.getPeripherals();
+        if (peripheralsConfiguration != null) {
+            PeripheralsConfigurationProfile peripheralsConfigurationProfile = peripheralsConfiguration.getProfile();
+            if (peripheralsConfigurationProfile != null) {
+                addStatisticsParameterWithDropdown(statistics, controllableProperties, PERIPHERALS_CAMERA_PROFILE_CONTROL, peripheralsConfigurationProfile.getCameras(), valuespace);
+                addStatisticsParameterWithDropdown(statistics, controllableProperties, PERIPHERALS_CONTROL_SYSTEM_PROFILE_CONTROL, peripheralsConfigurationProfile.getControlSystems(), valuespace);
+                addStatisticsParameterWithDropdown(statistics, controllableProperties, PERIPHERALS_TOUCH_PANEL_PROFILE_CONTROL, peripheralsConfigurationProfile.getTouchPanels(), valuespace);
+            }
+            PeripheralsPairing peripheralsPairing = peripheralsConfiguration.getPairing();
+            if (peripheralsPairing != null) {
+                CiscoTouchPanels ciscoTouchPanels = peripheralsPairing.getCiscoTouchPanels();
+                if (ciscoTouchPanels != null) {
+                    addStatisticsParameterWithDropdown(statistics, controllableProperties, PERIPHERALS_TOUCH_PANEL_REMOTE_PAIRING, ciscoTouchPanels.getRemotePairing(), valuespace);
+                }
+            }
+        }
+    }
+
+    /**
+     * Merge 2 strings together with a separator in between. If 1st string is empty or null - 2nd string is returned.
+     *
+     * @param s1        first string to add to merge result
+     * @param s2        second string to add to merge result
+     * @param separator to separate 2 substrings
+     * @return {@link String} merge result with separator
```

Comment:
Since there is a javadoc for this method, and the method is new in 1.1, you can add `@since 1.1`